ui/tests/acceptance/reset-password-test.js

```diff
@@ -71,37 +69,4 @@ module('Acceptance | reset password', function (hooks) {
     assert.dom('[data-test-flash-message]').hasText(`Success ${SUCCESS_MESSAGE}`);
     assert.dom('[data-test-input="reset-password"]').hasValue('', 'Resets input after save');
   });
-
-  test('allows password reset for userpass users logged in via tab', async function (assert) {
-    await login();
-    await runCmd([
-      mountAuthCmd('userpass', this.userpass),
-      `write sys/auth/${this.userpass}/tune listing_visibility="unauth"`,
-      createPolicyCmd(this.userpass, this.policy),
-      `write auth/${this.userpass}/users/reset-me password=password token_policies=${this.userpass}`,
-    ]);
-    await logout();
-    await click(`[data-test-auth-method="${this.userpass}"] a`);
-    await fillIn(GENERAL.inputByAttr('username'), 'reset-me');
-    await fillIn(GENERAL.inputByAttr('password'), 'password');
-    await click(AUTH_FORM.login);
-
-    await click('[data-test-user-menu-trigger]');
-    await click('[data-test-user-menu-item="reset-password"]');
-
-    assert.strictEqual(currentURL(), '/vault/access/reset-password', 'links to password reset');
-    assert
-      .dom('[data-test-current-user-banner]')
-      .hasText(
-        `You are updating the password for reset-me on the ${this.userpass} auth mount.`,
-        'shows correct banner text'
-      );
-
-    assert.dom('[data-test-title]').hasText('Reset password', 'page title');
-    await fillIn('[data-test-input="reset-password"]', 'newpassword');
-    await click('[data-test-reset-password-save]');
-    await waitFor('[data-test-flash-message]');
-    assert.dom('[data-test-flash-message]').hasText(`Success ${SUCCESS_MESSAGE}`);
-    assert.dom('[data-test-input="reset-password"]').hasValue('', 'Resets input after save');
-  });
```

Comment:
Is this no longer relevant?

Reply:
Good question! I meant to leave a comment. Now tabs vs dropdown are just a stylistic difference but functionally work the same, so there's no need to test logging in from a tab. 

Plus configuring an auth method with `listing_visibility="unauth"` messes up the test server state since any tests that run after fail to `login()` since the tab view shows instead of the default login form. It felt more reliable to remove any logic that actually configured methods with that setting than having to check the login form state for each test or make sure that the appropriate cleanup (deleting the mount) happened

Reply:
Sounds good thanks for the context!